blazar/enforcement/filters/external_service_filter.py

```diff
@@ -97,22 +97,22 @@ def post(self, path, body):
 
     def check_create(self, context, lease_values):
         if self.external_service_endpoint:
-            path = '/v1/check-create'
+            path = '/check-create/'
```

Comment:
I wonder if this should actually be configurable. Not all servers will have this weird trailing slash thing that Django has.

I don't think this code is upstreamed yet. @Mark-Powers wdyt about expanding the options slightly and renaming this one?

```
[enforcement]
# If only this is defined, Blazar will assume it can find "/check-create", "/check-update", "/on-end"
external_service = "https://127.0.0.1:4000/v1"
# But endpoints can individually be overridden as well w/ absolute URL
external_service_check_create = "https://127.0.0.1:4000/v1/check-create/
external_service_check_update = "..."
external_service_on_end = "..."
```

Reply:
Yes, it makes sense to have these options. It is not upstream yet.